addons/resource.language.en_gb/resources/strings.po

```diff
@@ -9742,9 +9742,10 @@ msgctxt "#19179"
 msgid "Deleted"
 msgstr ""
 
+#. Settings -> Interface -> Other -> Startup window
 #: system/settings/settings.xml
 msgctxt "#19180"
-msgid "TV"
+msgid "TV channels"
```

Comment:
Are you sure can change this string without negative impact on how it was used before?